### ISSUE TITLE
How can I use rover when module sources are in custom locations

### ISSUE THREAD

**Issue by carlosdoliveira** [im2nguyen/rover#17]
For example I have the following folder structure
```
./../../../
├── README.md
├── modules
│   └── vnet
├── products
│   ├── product1
│   │   ├── main.tf
│   │   ├── variables.tf
│   │   ├── outputs.tf
│   ├── product2
│   │   ├── main.tf
│   │   ├── variables.tf
│   │   ├── outputs.tf
```

Is there any way where I can run rover to consume the modules from the "modules" directory and initialize terraform from "product1" folder?

**Comment by NeilDavisNPS** [comment]
I think I have the same issue.. my modules are located elsewhere and when I run the docker command, it complains with:

`Unable to evaluate directory symlink: lstat ../../../../modules: no such file
or directory


Error: Failed to read module directory

Module directory  does not exist or cannot be read.`

**Comment by im2nguyen** [comment]
Similar to https://github.com/im2nguyen/rover/issues/30. 🤞 that once https://github.com/im2nguyen/rover/issues/20 is resolved, it'll resolve this issue. Hoping to get it pushed by end of week 😄

**Comment by im2nguyen** [comment]
https://github.com/im2nguyen/rover/pull/32 will "address" this. For this scenario, you will need to generate a plan then run rover against the plan file 😄 

```
$ rover -planPath "plan.out"
```

**Comment by im2nguyen** [comment]
Just merged #32, so this _should_ work.

If it works, can you 👍   this comment? Once there's enough 👍 s, I'll close this issue.

Please let me know if you have any feedback, thanks!

You can test this workflow by:

1. Go to `example/random-test` and run the following to generate a plan output.

    ```
    terraform plan -out plan.out
    ```

2.  Then, in the same directory, run the following command to force rover to use the plan file you specified

    ```
    docker run --rm -it -p 9000:9000 -v $(pwd):/src im2nguyen/rover -planPath plan.out
    ```

    To force `rover` to generate a visualization using the plan file only (no parsing the configuration), run the following command:

    ```
    docker run --rm -it -p 9000:9000 -v $(pwd):/src im2nguyen/rover -planPath plan.out -tfConfigExists=false
    ```

    You should see something like the following. Notice that there are no more file names in the visualization

    ![image](https://user-images.githubusercontent.com/7556325/135082937-94b61a78-0717-445f-b0e3-7b0b3ccf6cac.png)

**Comment by carlosdoliveira** [comment]
I'll test it out here and will update the feedbacks here!! Thanks for the help so far!
